### DISCUSSION TITLE
How can i make Files non translatable?

### DISCUSSION THREAD

**Discussion by ghost** [area17/twill#1690] [Q&A] [answered]
It seems like files are always translated, when setting the translated property to false in the form or setting 'translated_form_fields'  to false in the twill file_library doesn't work. 
It seems like this should be an option. Is there a way to solve this?
Thanks!

**Comment by haringsrob** [comment]
Hey @vinceinfluo,

What exactly do you want to accomplish? To have for example a different image per translation?

**Comment by ghost** [comment]
Hi Rob,

Thanks for your answer, I'm not talking about media, media can be set as non-translatable. But for files (non-media files, just like pdf, zip, etc) it seems impossible to set those as non-translatable. 
Meaning that if I want to make a file available I have to attach the file to my page for each language.

<img width="1022" alt="Screenshot 2022-06-24 at 13 25 52" src="https://user-images.githubusercontent.com/22447612/175525263-114b67a1-9f4c-47ee-abaa-754c732ec3e3.png">

**Comment by haringsrob** [comment]
They are indeed currently not not-translatable. I created an issue but I feel this will not happen too soon. Maybe in 3.x we can add it.

**Comment by haringsrob** [comment, marked as answer]
What you could do currently is to fallback to the english version wherever you are using the file.

**Reply by ifox** [reply]
@vinceinfluo This is actually the default behavior of the `file` and `files` methods on the `HasFiles` trait, so technically your CMS users don't need to select a file per language.
